doc/rest_api/snippet_sharing/api-description.apib

```diff
@@ -0,0 +1,587 @@
+
+## List database entries [GET /database/{organization}/{application}/{scope}{?offset}{?rows}{?sort}{?sortby}{?filter}{?filterby}]
+
+Returns a list of database entries. The result can be adjusted by additional, but optional parameters.
+
++ Parameters
+    + organization (string, optional) - Only search within a given `organization` for entries
+    + application (string, optional) - Only search within a given `application` for entries
+    + scope (string, optional) - Only search within a given `scope` of an `application` for entries
+    + offset (number, optional) - How many entries to skip for the output, can be used for pagination.
+        + Default: `0`
+    + rows (number, optional) - How many entries will be added to the output at max.
+        + Default: `200`
+    + sort (enum[string], optional) - How to sort the entries for the output.
+        + Default: `asc`
+        + Members
+            + `asc`
+            + `desc`
+    + sortby (enum[string], optional) - What criteria should be used for sorting
+        + Default: `key`
+        + Members
+            + `key` - Absolute entry path
+            + `title` - Entry title
+            + `created_at` - Creation date
+            + `author` - Authors name
+            + `organization` - Organization to which the application of the configuration belongs to
+            + `application` - Application for which the configuration is meant
+            + `scope` - Application internal scope
+            + `slug` - Entry specific unique slug
+    + filter (string, optional) - Only entries containing the search string in either their path, title, description, author or tags will be added to the output.
+    + filterby (enum[string], optional) - On what fields the filter should be applied to (i.e. what fields to involve in search)
+        + Default: `all`
+        + Members
+            + `all` - All fields: key, title, description, author and tags
+            + `key` - Only the key
+            + `title` - Only the title
+            + `description` - Only the description
+            + `author` - Only the authors name
+            + `tags` - Only tags
+
++ Response 200 (application/json)
+    + Attributes (Entry List)
+
+
+## Get details for entry [GET /database/{organization}/{application}/{scope}/{slug}{?raw}]
+
+Returns detailed information about the entry with the specified key (path). If the optional parameter `raw` is supplied, a raw configuration is returned instead of the normal response.
+
++ Parameters
+    + organization (string, required) - The `organization` who built the `application` for which the configuration of the entry is meant
+    + application (string, required) - The `application` to which the configuration of the entry belongs to
+    + scope (string, required) - The `application` specific `scope` to which the configuration of the entry belongs to
+    + slug (string, required) - The unique identifier of the requested entry
```

Comment:
Slug is only required to be unique within the namespace created by org, app and scope. Theoretically the slug could also be created from the snippet title - maybe this would be a good change?